src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -0,0 +1,166 @@
+import {
+  IPIAIntake,
+  IPIAIntakeResponse,
+} from '../../types/interfaces/pia-intake.interface';
+import { faArrowDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { dateToString } from '../../utils/date';
+import messages from './messages';
+
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { HttpRequest } from '../../utils/http-request.util';
+import { API_ROUTES } from '../../constant/apiRoutes';
+import { routes } from '../../constant/routes';
+
+const PIADetailPage = () => {
+  // https://github.com/microsoft/TypeScript/issues/48949
+  // workaround
+  const win: Window = window;
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const [pia, setPia] = useState<any>({});
+  useEffect(() => {
+    (async () => {
+      try {
+        // Actually perform fetch
+        const result = (
+          await HttpRequest.get<IPIAIntakeResponse>(
+            API_ROUTES.GET_PIA_INTAKE.replace(':id', `${id}`),
+          )
+        ).data;
+        setPia(result);
+      } catch (e: any) {
```

Comment:
Can we be more specific with our type?

Reply:
this one can not define a specific type, in ts the catch part only allow any or unknown type, it will address in my pr